### DISCUSSION TITLE
Spatial Viewer support for Oracle SE

### DISCUSSION THREAD

**Discussion by dntichy** [dbeaver/dbeaver#12958] [DBeaver Development]
Hi guys, I like this Spatial viewer very much but unfortunately Oracle SE is not supported according to
https://github.com/dbeaver/dbeaver/wiki/Working-with-Spatial-GIS-data

Are you considering support for Oracle SE ? 
Thanks

**Comment by LonwoLonwo** [comment]
Hello @dntichy 
DBeaver supports spatial data in the [DBeaver EE version](https://dbeaver.com/).

![2021-06-23 18_04_43-Working with Spatial GIS data · dbeaver_dbeaver Wiki — Mozilla Firefox](https://user-images.githubusercontent.com/45152336/123121513-e4bb9100-d44d-11eb-9e9c-98d98aa9ba19.png)

You can try it with [a trial license](https://dbeaver.com/trial/).

**Reply by dntichy** [reply]
hh, thx, I misunderstood that a bit. 
I am currently on trial, I like that :)
